adding feature json

lukaklanjsek opened the pull request:
this is me trying to add feature "json" to an existing task. I did however leave the main file "procedure_example" to still work with the command line input.  Depending on the task it can be quickly changed into something more apropriate (input) if needed.

lukaklanjsek committed:
adding feature json

this is me trying to add feature "json" to an existing task. I did however leave the main file "procedure_example" to still work with the command line input.  Depending on the task it can be quickly changed into something more apropriate (input) if needed.

lukaklanjsek committed:
adding feature json

this is me trying to add feature "json" to an existing task. I did however leave the main file "procedure_example" to still work with the command line input.  Depending on the task it can be quickly changed into something more apropriate (input) if needed.

matevz commented on example.json:
```diff
@@ -0,0 +1,52 @@
+{
```
This should be a list and not a dictionary. Dictionaries are not ordered by default.

```suggestion
[
```

matevz commented on play_around.py:
```diff
@@ -202,7 +202,7 @@
 test_dict = {"Luka" : 5, "Matija" : 2, "Urša" : 1, "Leopold" : 3, "Karla" : 4}
 sorted_test_dict = dict(sorted(test_dict.items(), key=lambda item: item[1], reverse=False))
 print(sorted_test_dict)    # okay this works now finally yay big happy
-"""
```
Triple quotes should be used for documentation (pydoc) and not for commenting out lines of code. Use `#` for doing that.

lukaklanjsek replied:
I intend to remove the file offline and delete it from github. It is only a testing area.

matevz reviewed (commented):
Needs some cleanups.

lukaklanjsek committed:
updated readme to current state of affairs

lukaklanjsek committed:
adding feature json

this is me trying to add feature "json" to an existing task. I did however leave the main file "procedure_example" to still work with the command line input.  Depending on the task it can be quickly changed into something more apropriate (input) if needed.

lukaklanjsek committed:
Merge remote-tracking branch 'origin/lukaklanjsek/feature/json' into lukaklanjsek/feature/json

# Conflicts:
#	example.json
#	play_around.py
#	procedure_example.py
